crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -267,29 +267,44 @@ pub fn create_test_provider() -> JsonRpcClient<HttpTransport> {
 }
 
 #[must_use]
-pub fn duplicate_directory_with_salt(src_path: String, to_be_salted: &str, salt: &str) -> TempDir {
+pub fn duplicate_contract_directory_with_salt(
+    src_path: String,
+    to_be_salted: &str,
+    salt: &str,
+) -> TempDir {
+    duplicate_directory_and_salt_file(src_path, None, to_be_salted, "src/lib.cairo", salt)
+}
+
+#[must_use]
+pub fn duplicate_directory_and_salt_file(
+    src_path: String,
+    dst_path: Option<String>,
+    code_to_be_salted: &str,
+    file_to_be_salted: &str,
+    salt: &str,
+) -> TempDir {
     let src_dir = Utf8PathBuf::from(src_path);
-    let temp_dir = TempDir::new().expect("Unable to create a temporary directory");
+    let temp_dir = match dst_path {
+        Some(dst_path) => TempDir::new_in(dst_path),
```

Comment:
https://github.com/foundry-rs/starknet-foundry/pull/1358#discussion_r1472518263

Reply:
if you're not moving tempdirs from specified paths, let's maybe do:

```
    let temp_dir = match dst_path {
        Some(dst_path) => TempDir::new_in(dst_path).expect("Unable to create a temporary directory in the specified path"),
        None => TempDir::new().expect("Unable to create a temporary directory"),
    };
```

?

Reply:
Maybe this:
```rust
let temp_dir = match dst_path {
        Some(dst_path) => TempDir::new_in(dst_path),
        None => TempDir::new(),
    }.expect("Unable to create a temporary directory");
```

Reply:
done